packages/ts-moose-lib/src/consumption-apis/helpers.ts

```diff
@@ -32,6 +32,7 @@ export const mapToClickHouseType = (value: Value) => {
   if (value instanceof Date) return "DateTime";
   if (Array.isArray(value)) {
     const [type, _] = value;
+    if (type === "Date") return "Date";
```

Comment:
Why is this in the case of an `Array` and not at the same hierarchical level as the case for `DateTime`